modules/nested-accordion/assets/js/frontend/handlers/nested-accordion.js

```diff
@@ -34,4 +36,74 @@ export default class NestedAccordion extends Base {
 			$items[ index ].appendChild( element );
 		} );
 	}
+
+	applyDefaultStateCondition() {
+		if ( ! this.elements ) {
+			return;
+		}
+
+		const accordionItems = this.elements.$items,
+			defaultState = this.getDefaultStateCondition(),
+			stateFirstExpanded = 'first_expended';
+
+		if ( stateFirstExpanded === defaultState ) {
+			accordionItems[ 0 ].setAttribute( 'open', '' );
+		} else {
+			accordionItems.each( ( _, item ) => item.removeAttribute( 'open' ) );
+		}
+	}
+
+	getDefaultStateCondition() {
+		return elementorFrontend.utils.controls.getControlValue( this.getElementSettings(), 'default_state', '' );
+	}
+
+	bindEvents() {
+		elementorFrontend.elements.$window.on( 'resize', this.applyDefaultStateCondition.bind( this ) );
+		this.bindAnimationListeners();
+	}
+
+	unbindEvents() {
+		elementorFrontend.elements.$window.off( 'resize' );
+		this.removeAnimationListeners();
+	}
+
+	bindAnimationListeners() {
+		const { $items } = this.getDefaultElements();
+
+		$items.each( ( _, element ) => {
+			element.addEventListener( 'click', ( e ) => {
+				if ( element.hasAttribute( 'open' ) ) {
```

Comment:
Maybe extract the inner condition to a separate function.
I see we are using the code block twice
As well the code will be cleaner by avoiding nested conditions.